### ISSUE TITLE
Curate a proper set of genes that much represent a gold-standard for early pipeline checks

### ISSUE THREAD

**Issue by mkarmona** [opentargets/platform#516]
Having a list of genes that represents a gold-standard for a pipeline test run will improve checks and speedup diagnoses. An initial set can come from this poreviously curated list

```
ENSG00000169194
ENSG00000073756
ENSG00000073605
ENSG00000110324
ENSG00000085978
ENSG00000105397
ENSG00000142192
ENSG00000080815
ENSG00000164885
ENSG00000147889
ENSG00000157764
ENSG00000134460
```

**Comment by ElaineMcA** [comment]
Discussed at team meeting on 08Apr2019.  Recommendation to start with list of top 50 genes that people may view.

- extending assignment to Andrew also.

**Comment by deniseOme** [comment]
@iandunham, following your suggestions I was thinking: is there a privacy issue for looking into the logs of what people search for?

In addition to Eliseo's and @mkarmona's list we may also want to include the [10 greatest hits of the human genome](https://www.nature.com/articles/d41586-017-07291-9) (by Nature).

ENSG00000141510
ENSG00000232810
ENSG00000146648
ENSG00000112715
ENSG00000130203
ENSG00000136244
ENSG00000105329
ENSG00000177000
ENSG00000091831 (ESR1 aka @andrewhercules' target)
ENSG00000142208

**Comment by iandunham** [comment]
Not if we use them for improving the platform i.e. testing, and don't make them public. However using the list you identified wopuld also be useful - we could mix in some others

**Comment by andrewhercules** [comment]
Just to allay any concerns, we anticipated the need to cover these types of activities. And so we have a specific line in [our privacy notice](https://www.ebi.ac.uk/data-protection/privacy-notice/open-targets) that allows us to analyse analytics and search data to make website improvements and for pipeline data QC:

``We do not analyse the nature of any specific queries (target or disease names) processed via the website except for the purposes of improving the site.``

As promised in yesterday's Core Platform team meeting, I have created [a spreadsheet hosted on the team Google Drive](https://docs.google.com/spreadsheets/d/1wdr7zJuuh8KiYxjYgfrfJbwU55OU1wDFrElTv8R2WMg/edit#gid=1648714034) with the Ensembl IDs and HGNC symbols for the 50 most-searched targets. I have also included a separate tab with the ten most studied genes that are listed in the Nature paper referenced by @deniseOme.
